SignalMessaging/categories/UIColor+OWS.m

```diff
@@ -79,7 +79,7 @@ + (UIColor *)ows_reminderDarkYellowColor
 
 + (UIColor *)ows_destructiveRedColor
 {
-    return [UIColor colorWithRed:0.98639106750488281 green:0.10408364236354828 blue:0.33135244250297546 alpha:1.f];
+    return [UIColor colorWithRed:255.f / 255.f green:38.f / 255.f blue:31.f / 255.f alpha:1.0f];
```

Comment:
For consistency, can we converge on using the hex constructor whenever reasonable?

Reply:
I'd like to use hex as a default, and I'll change it here.

However for non-palette colors delivered with Zeplin, I'm inclined to use this representation so it can be easily verified against the design.